### DISCUSSION TITLE
Remove Ring Of Valor from pool of possible arena maps.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#7077] [Suggestions]
I suggest removing the Ring of Valor from the pool of possible arena maps. This arena is widely disliked within the community due to its poor design and gameplay issues. Its mechanics, like the elevators and line-of-sight obstacles, create frustrating and unbalanced matches. Reflecting this consensus, most private servers have already removed the Ring of Valor, and it was also excluded from Wrath Classic by Blizzard. Removing it from Chromiecraft would enhance the overall PvP experience and align with community preferences.

**Comment by heyitsbench** [comment]
But what if I love Ring of Valor?

**Comment by Never6** [comment]
hard agree

**Comment by TotallyNotMyAccount** [comment]
Trash arena, get rid of it.

**Comment by Nyeriah** [comment]
Hello,

Thanks for your suggestion. It should be removed from the available arenas pool in the next update.

Regards.

**Comment by UndoUreche** [comment]
it's still here!
